client/src/components/RelatedItems/YourOutfit.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import AddOutfit from './AddOutfit.jsx';
+import ProductCard from './ProductCard.jsx';
+
+function YourOutfit({outfitProductsInfo}) {
+  return (
+    <div>
+      <AddOutfit />
+      {outfitProductsInfo.map(outfitProductInfo => {
```

Comment:
Because outfitProductInfo is empty now, it's not rendering anything. 
The documentation says: _By default, this list should contain no products within it._  
So I feel like that's the correct result for a start.
OutfitProduct is not stored in the database, and I need to come up with a solution for that. (And that's why I'm also skipping that component for now).